packages/grafana-data/src/themes/colorManipulator.test.ts

```diff
@@ -260,7 +260,7 @@ describe('utils/colorManipulator', () => {
 
   describe('alpha', () => {
     it('converts an rgb color to an rgba color with the value provided', () => {
-      expect(alpha('rgb(1, 2, 3)', 0.4)).toEqual('rgba(1, 2, 3, 0.4)');
+      expect(alpha('rgb(1, 2, 3)', 0.4)).toEqual('rgb(1, 2, 3, 0.4)');
```

Comment:
This case does not look correct

Reply:
it's spec-compliant: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()#accepts_alpha_value